doc/userguide/rules/dhcp-keywords.rst

```diff
@@ -25,6 +27,8 @@ dhcp.rebinding_time
 
 DHCP rebinding time (integer).
 
+dhcp.rebinding_time uses an, :ref:` unsigned 64-bits integer <rules-integer-keywords>`.
```

Comment:
```suggestion
dhcp.rebinding_time uses an, :ref:`unsigned 64-bits integer <rules-integer-keywords>`.
```

^edit necessary for the `ref` to actually become a link

Reply:
ok